BMS_Receiver/BMS_Receiver.c

```diff
@@ -0,0 +1,144 @@
+/******************************************************************************/
+/*!
+***     \file        BMS_Receiver.c
+***     \author      Abinesh KanjiKovil Thatchinamoorthy 
+***
+***
+***     \brief       Source for BMS Receiver implementation
+***
+*\n*/
+/*****************************************************************************/
+
+/* Inclusions */
+#include "BMS_Receiver.h"
+
+/* Declarations */
+bmsReceiver_t bmsReceiverData_s = {0};
+
+/*---------------------------------------------------------------------------*/
+/*     FUNCTION:    readBMSdata_v
+ */
+/*!    \brief       Reads the BMS data and stores it
```

Comment:
This comment doesn't say anything more than the function name. Does it read ALL the samples? Or only one? Naming the function like `readBMSdataSample` would save a lot of time in reading it.